index.html

```diff
@@ -0,0 +1,134 @@
+                fill="#505F79"
+              />
+            </svg>
+          </a>
+
+          <a href="https://twitter.com/binhussens">
+            <svg
+              width="21"
+              height="20"
+              viewBox="0 0 21 20"
+              fill="none"
+              xmlns="http://www.w3.org/2000/svg"
+            >
+              <path
+                d="M20.1753 0.394423C19.842 -0.00424693 19.2753 -0.103914 18.842 0.195088L16.9086 1.52399C16.0086 0.560536 14.742 0.0289756 13.3753 0.0289756C10.6753 0.0289756 8.44198 2.22166 8.44198 4.94591C8.44198 5.34458 8.47532 5.71002 8.57532 6.07547C6.24198 5.57713 4.84198 3.88279 4.30865 0.826316C4.24198 0.427646 3.94198 0.128643 3.54198 0.0289756C3.17532 -0.070692 2.77532 0.0954206 2.54198 0.394423C2.47532 0.460868 -3.02468 8.10205 4.94198 14.5804C4.17532 16.2083 1.87532 17.6037 0.975315 18.0688C0.541982 18.2681 0.341982 18.7332 0.441982 19.1984C0.541982 19.6635 0.941982 19.9625 1.40865 19.9625H3.14198C13.2753 19.9625 17.8753 14.9459 17.9753 3.71667L20.0753 1.72332C20.4753 1.3911 20.5086 0.793093 20.1753 0.394423Z"
+                fill="#505F79"
+              />
+            </svg>
+          </a>
+
+          <a href="">
+            <svg
+              width="21"
+              height="16"
+              viewBox="0 0 21 16"
+              fill="none"
+              xmlns="http://www.w3.org/2000/svg"
+            >
+              <path
+                d="M18.8208 1.88042L20.4167 0.337684V0H14.8883L10.9483 9.90989L6.46582 0H0.669156V0.337684L2.53332 2.60547C2.71499 2.77305 2.80999 3.01811 2.78582 3.26484V12.1768C2.84332 12.4977 2.73999 12.8278 2.51666 13.0611L0.416656 15.6328V15.9663H6.37082V15.6286L4.27082 13.0611C4.04332 12.8269 3.93582 12.5027 3.98166 12.1768V4.46821L9.20832 15.9705H9.81582L14.31 4.46821V13.6312C14.31 13.8728 14.31 13.9225 14.1533 14.0808L12.5367 15.6615V16H20.38V15.6623L18.8217 14.1204C18.685 14.016 18.6142 13.8408 18.6433 13.6707V2.33011C18.6142 2.15916 18.6842 1.984 18.8208 1.88042Z"
+                fill="#505F79"
+              />
+            </svg>
+          </a>
+        </div>
```

Comment:
- [ ] your social media icons are displayed please make sure to fix that and also they should in wrapped them in `ul` tag and each one in `li` tag and the icon should be `<a>` tag
- Yours 
![image](https://user-images.githubusercontent.com/61790587/176547486-953359c1-08a6-40ce-b150-03831eb0fa18.png)
- Figma 
![image](https://user-images.githubusercontent.com/61790587/176547501-867e8f78-5427-4dbb-a941-12fd59ef0864.png)

Reply:
Thank you @sumairq for your feedback, I will Modify It.